From: Dan Streetman <ddstreet@canonical.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1529152
Forwarded: not-needed

'systemctl show' in Bionic does not include upstream commit
eda193578effbc3cee0f6d56ade52434778737c9, which allows the property value
to be optionally included in the property string.  So, the backport to
Bionic needs to strip the =value from each property name before asking
'systemctl show' to print them.

--- systemd-237.orig/shell-completion/bash/systemctl.in
+++ systemd-237/shell-completion/bash/systemctl.in
@@ -42,13 +42,14 @@ __contains_word () {
 __filter_units_by_properties () {
         local mode=$1 properties=$2; shift 2
         local units=("$@")
-        local props i p n
+        local propertynames props i p n
         local names= count=0
 
         IFS=$',' read -r -a p < <(echo "Names,$properties")
+        IFS=$',' propertynames=("${p[*]//=*/}")
         n=${#p[*]}
         readarray -t props < \
-                  <(__systemctl $mode show --property "Names,$properties" -- "${units[@]}")
+                  <(__systemctl $mode show --property "Names,$propertynames" -- "${units[@]}")
 
         for ((i=0; i < ${#props[*]}; i++)); do
                 if [[ -z ${props[i]} ]]; then
